libr/core/project.c

```diff
@@ -87,16 +87,18 @@ R_API bool r_core_is_project(RCore *core, const char *name) {
 }
 
 R_API int r_core_project_cat(RCore *core, const char *name) {
```

Comment:
I think it should return bool. The command handler who calls this function is the one that must change the return code